Assets/ZSSRichTextEditor.js

```diff
@@ -1063,7 +1063,8 @@ ZSSEditor.markVideoUploadDone = function(videoNodeIdentifier) {
     }
     var joinedArguments = ZSSEditor.getJoinedFocusedFieldIdAndCaretArguments();
     ZSSEditor.callback("callback-input", joinedArguments);
-    this.sendVideoReplacedCallback(videoNodeIdentifier);
+    var thisObj = this;
+    setTimeout(function() { thisObj.sendVideoReplacedCallback(videoNodeIdentifier);}, 500);
```

Comment:
@SergioEstevao Could you put a descriptive comment in here to explain why this delay is being introduced?